### ISSUE TITLE
Bug: The progress cannot be killed by Accessibility

### ISSUE THREAD

**Issue by AnrayAnn** [d4rken/sdmaid-public#5075]
### What happened?
When the Accessibility turns on, SD Maid cannot kill the App's progress successfully. Instead, SD Maid will keep cycling in the same App Info

### What did you expect to happen?
Kill the progress and keep on cleaning

### How can I reproduce the issue?
Open the SD Maid and turn on the Accessibility. Then use SD Maid to clean.

### Comments
It's a very awful bug so I hope that it can be solved rapidly.
### SD Maid
Version: `5.1.4.13 (5010413)`
Pro: `isPro=true || isComplete=true (unl=true, iap=false, acc=false)`

### Device
Root: `RootContext(rootState=Root(state=UNAVAILABLE)`
Fingerprint: `OnePlus/OnePlus9Pro_CH/OnePlus9Pro:11/RKQ1.201105.002/1626020145774:user/release-keys`

**Comment by d4rken** [comment]
Please upgrade to the latest version, if it still does not work, provide a debug log.

**Comment by AnrayAnn** [comment]
https://1drv.ms/t/s!Ane6OdRbS9m1hDoBKh8A3UdltJIT

I upload the log in Onedrive
